wagtail/contrib/search_promotions/templates/wagtailsearchpromotions/index.html

```diff
@@ -1,6 +1,5 @@
-{% extends "wagtailadmin/base.html" %}
+{% extends "wagtailadmin/generic/index.html" %}
 {% load i18n %}
-{% block titletag %}{% trans "Search Terms" %}{% endblock %}
 
 {% block content %}
     {% trans "Promoted search results" as sp_title_str %}
```

Comment:
We need to also change the `search_results_url` here, although ideally we should try whether we can completely remove this block override (or override sub-blocks in `generic/index.html`) instead. Then, setting `_show_breadcrumbs = True` on the view class would allow this view to use the new designs.

Reply:
@laymonage Sorry i forgot to update this URL :)

I attempted removing the block override from `wagtailsearchpromotions/index.py`, and here's the updated UI:

![image](https://github.com/wagtail/wagtail/assets/111359305/0d6c2de3-b64b-479c-aa5a-ca54c2980ef7)

Is it fine?